src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Consider removing the empty lines (lines 3-5) before the class declaration to adhere to the coding best practices.